OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -59,6 +59,10 @@ const uint8_t MCP_COUNT             = sizeof(MCP_I2C_ADDRESS);
 // Each bit corresponds to an MCP found on the IC2 bus
 uint8_t g_mcps_found = 0;
 
+// flags ports when the config has changed from/to SECURITY 
+// (bitwise coded per port, 1 = SECURITY, 0 = other)
+uint32_t g_port_new_config = 0;

```

Comment:
Since this is specifically related to `SECURITY` inputs, could we rename this `g_security_ports`, or similar?